### PR TITLE
[UI]Public interface cleanup

### PR DESCRIPTION
**Pull request by ipavlidakis**
### 🔗 Issue Links

_Provide all JIRA tickets and/or GitHub issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

|  Before  |  After  |
| -------- | ------- |
|  `img`   |  `img`  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should receive manual QA
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (Docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: Sources/StreamVideoSwiftUI/Appearance.swift [hunk 1]
```diff
@@ -51,4 +51,44 @@ extension InjectedValues {
             Self[AppearanceKey.self] = newValue
         }
     }
+
+    /// Provides access to the `Colors` instance.
+    public var colors: Colors {
```
Comment (martinmitrevski) [inline Sources/StreamVideoSwiftUI/Appearance.swift:56]: wow, we didn't have these exposed?

File: Sources/StreamVideoSwiftUI/CallView/CornerDraggableView.swift [hunk 2]
```diff
@@ -45,14 +52,7 @@ public struct CornerDraggableView<Content: View>: View {
     public var body: some View {
         content(availableFrame)
             .overlay(
-                // SwiftUI seems to remove any interaction of views that not
```
Comment (martinmitrevski) [inline Sources/StreamVideoSwiftUI/CallView/CornerDraggableView.swift:48]: why do we remove these? There's some useful info here.

File: Sources/StreamVideoSwiftUI/Appearance.swift (hunk 1 above)
Comment (martinmitrevski) [reply Sources/StreamVideoSwiftUI/Appearance.swift:56]: nvm, found them in another file

File: Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift [hunk 3]
```diff
@@ -188,11 +214,11 @@ extension VideoRenderer {
             )
             add(track: track)
             DispatchQueue.global(qos: .userInteractive).asyncAfter(deadline: .now() + 0.01) { [weak self] in
-                guard let self else { return }
+                guard let self = self else { return }
                 let prev = participant.trackSize
-                if let viewSize, prev != viewSize {
+                if let viewSize = self.viewSize, prev != viewSize {
```
Comment (martinmitrevski) [inline Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift:219]: Is this needed for iOS 14 issues?

**Review by martinmitrevski — COMMENTED** [review]
Looks good 👌 Not sure about the deleted comments in `CornerDragableView`, probably good to keep them?

File: Sources/StreamVideoSwiftUI/CallView/CornerDraggableView.swift (hunk 2 above)
Comment (ipavlidakis) [reply Sources/StreamVideoSwiftUI/CallView/CornerDraggableView.swift:48]: My mistake. I brought them back

File: Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift (hunk 3 above)
Comment (ipavlidakis) [reply Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift:219]: I think it was one of the things i tried. I'll remove to keep the diff only to meaninful changes

**Review by martinmitrevski — APPROVED** [review]

**Pull request merged**